load/loader.py

```diff
@@ -102,7 +102,16 @@ def load_voting_history(opts, conf):
         '-param:reportFileDir={}'.format(opts.input_file),
         '-param:reporterKey={}'.format(opts.reporter_key)
     ])
-
+    elif opts.state == 'ny':
+        fl_path = os.path.dirname(os.path.join(conf['data_path'],
```

Comment:
`fl_path` should be `ny_path` or a more generic name just to make things easier to read